shared-ui/build.gradle.kts

```diff
@@ -0,0 +1,45 @@
+@Suppress("DSL_SCOPE_VIOLATION") // TODO: Remove once KTIJ-19369 is fixed
```

Comment:
Same here, we can remove the suppression 🎉

Reply:
done. This is in the new module template weirdly

Reply:
Oh is it?!? Damn. I'll file a bug.

Reply:
Can't find it in the Studio sources on the main branch, may be only in older versions :)